recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+{% set file_url = "http://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: scotch_{{ version }}.tar.gz
+  md5: {{ md5 }}
+  url: {{ file_url }}
+  patches:
+    - clock_gettime.patch  # [osx]
```

Comment:
This [one]( https://github.com/Homebrew/homebrew-science/blob/ce9c31eea5f42c0b894a8eb2a24bd17bc7880c7b/scotch.rb ). If so, I'm not seeing the patch. I did see a similarly named patch for Perl. In any event, I think we can drop this.